src/main/java/org/traccar/protocol/HuabaoProtocolDecoder.java

```diff
@@ -699,6 +699,14 @@ private Position decodeLocation(DeviceSession deviceSession, ByteBuf buf) {
                         }
                     }
                     break;
+                case 0xF6:
+                    buf.readUnsignedByte(); // data type
+                    int fieldMask = buf.readUnsignedByte();
+                    buf.readUnsignedShort(); // light
```

Comment:
Shouldn't light also be behind a flag?

Reply:
they also provide an excel which makes it easier to understand:

https://github.com/AOVX-Corporation/AOVX_Share/blob/main/Document/en/01_G%20series/AOVX_G%26A_Series%20JTT808%20Protocol.xlsx

![Screenshot 2024-08-24 at 22 21 43](https://github.com/user-attachments/assets/21584748-8a40-4327-bdb8-51322dcaacca)

Reply:
I don't think you answered my question.

Reply:
you are totally right